dts/arm/nordic/nrf51822.dtsi

```diff
@@ -153,3 +154,11 @@
 &nvic {
 	arm,num-irq-priority-bits = <2>;
 };
+
+&sw_pwm {
+	timer-instance = <1>;
+	timer-capture-compare-count = <4>;
+	clock-prescaler = <0>;
+	ppi-base = <7>;
+	gpiote-base = <0>;
+};
```

Comment:
I think `status = "ok";` should be added here. Then, there would be no need to add overlay files in most cases (only when for example `clock-prescaler` has to be modified).

Reply:
Agreed.